book.toml

```
@@ -13,3 +13,42 @@ command = "mdbook-mermaid"
 
 [output.html]
 additional-js = ["doc/mermaid.min.js", "doc/mermaid-init.js"]
+
+[output.linkcheck]
+# Should we check links on the internet? Enabling this option adds a
+# non-negligible performance impact
+follow-web-links = false
+
+# Are we allowed to link to files outside of the book's root directory? This
+# may help prevent linking to sensitive files (e.g. "../../../../etc/shadow")
+traverse-parent-directories = false
+
+# If necessary, you can exclude one or more links from being checked with a
+# list of regular expressions. The regex will be applied to the link href (i.e.
+# the `./index.html` in `[some page](./index.html)`) so it can be used to
+# ignore both web and filesystem links.
+#
+# Hint: you can use TOML's raw strings (single quote) to avoid needing to
+# escape things twice.
+exclude = [ 'google\.com' ]
```

Why would we need this exclusion?

```suggestion
exclude = [ ]
```

---

It was a default setting.